packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA ITI-39 Processor/destinationConnector-XCA Endpoint-responseTransformer-step-1-Case 1 - Success or PartialSuccess.js

```diff
@@ -61,9 +62,10 @@ if ('Success' == queryResponseCode.toString() || 'PartialSuccess' == queryRespon
 				// TODO 1350 Create a function to get the attributes using getObjectAttributes() - this is returning the whole file!
 				// https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/s3/S3Client.html#getObjectAttributes(software.amazon.awssdk.services.s3.model.GetObjectAttributesRequest)
 				var docExists = fileExistsOnS3(filePath.toString());
+        const url = "https://" + bucketName + ".s3."  + bucketRegion + ".amazonaws.com/" + filePath.toString();
 
-				attachment.fileName = fileName.toString();
-				attachment.url = filePath.toString();
+				attachment.fileName = filePath.toString();
+				attachment.url = url;
```

Comment:
nit: can store filePath.toString() in a const so we don't have to convert 3 times